src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -2071,4 +2074,12 @@ private function getCustomerMessageErrorMapping(Exception $exception): array
             ],
         ];
     }
+
+    private function isFeatureFlagIsEnabledForMultiShipment()
```

Comment:
```suggestion
    private function isFeatureFlagIsEnabledForMultiShipment(): bool
```

If that is the proper type